remedy.lic

```diff
@@ -56,6 +56,7 @@ class Remedy
   end
 
   def create_item
+    stow_hands
```

Comment:
You need to put `common-items` in the `custom_require` call; and put `include DRCI` at the top.